resources/views/layouts/default/flights/simbrief_aircraft.blade.php

```diff
@@ -14,6 +14,8 @@
         <option value="ZZZZZ">Please Select An Aircraft</option>
         @foreach($subfleets as $subfleet)
           @foreach($subfleet->aircraft as $ac)
+            @if(setting('pireps.only_aircraft_at_dpt_airport') && $flight->dpt_airport_id == $ac->airport_id || !setting('pireps.only_aircraft_at_dpt_airport') 
+            @endif
```

Comment:
This will not work ... @endif must be after the <option ...> Simply change the order ( 17 if line / 18 option line  / 19 endif line )